src/utils/local_file_mods.py

```diff
@@ -44,7 +44,7 @@ def write_local_csv(filepath: str, data: pd.DataFrame):
     # Open the file in write mode
     with open(filepath, "w", newline="\n", encoding="utf-8") as file:
         # Write dataframe to the file
-        data.to_csv(file, index=False)
+        data.to_csv(file, date_format="%Y-%m-%d %H:%M:%S.%f+00", index=False)
```

Comment:
Tested by add;
         data["timestamp"] = pd.to_datetime(data["timestamp"], format="%d/%m/%Y-%H:%M:%S")
         data["timestamp"] = data["timestamp"].dt.strftime("%m/%d/%Y-%H:%M:%S")

And got an empty data error - not sure if there's a better way to test this

Reply:
Many thanks for looking at this. I chose this date_format by looking at the `Description = "Datetime format = %Y-%m-%d %H:%M:%S.%f+00"` in  the `contributors_schema.toml`. I tested the output looked good in the full_responses output, and noted that it was the same format as in the original (old) version of the contributors csv given by the business area.

Reply:
Ran with and without and got expected output so good to merge this one now